Servers/UI/OJS.Servers.Ui/ClientApp/src/components/diff/Diff.tsx

```diff
@@ -53,7 +57,15 @@ const Diff = ({ expectedStr, actualStr } : ITestsRunDiffProps) => {
                   splitView
                   showDiffOnly
                   renderContent={highlightSyntax}
-                  styles={{ gutter: { minWidth: 'unset', height: 'inherit' } }}
+                  styles={{
+                      gutter: {
+                          minWidth: 'unset',
+                          height: 'inherit',
+                          backgroundColor: themeColors.baseColor500,
+                          color: themeColors.textColor,
+                      },
+                      diffContainer: { backgroundColor: themeColors.baseColor500, color: themeColors.textColor },
```

Comment:
Can you split these in separate rows like the above obj?

Reply:
done